src/agent/netlink/Cargo.toml

```diff
@@ -9,7 +9,7 @@ edition = "2018"
 [dependencies]
 libc = "0.2.58"
 nix = "0.17.0"
-protobuf = "2.6.1"
+protobuf = "2.8.1"
```

Comment:
Whis is this version different to the top-level and `protocols` `Cargo.toml` (`2.14.0`)?

Reply:
yes, you are right, it should be "2.14.0".